alcs-frontend/src/app/features/board/board.component.ts

```diff
@@ -489,7 +489,7 @@ export class BoardComponent implements OnInit, OnDestroy {
       data,
     });
 
-    dialogRef.afterClosed().subscribe((isDirty) => {
+    dialogRef.beforeClosed().subscribe((isDirty) => {
```

Comment:
is this an improvement, does not look like angular update

Reply:
This was from angular update, it was firing this after the dialog closed and we navigated to another page, bringing us right back to this page.